FreeRTOS-Plus/Demo/AWS/Jobs_Windows_Simulator/Jobs_Demo/DemoTasks/JobsDemoExample.c

```diff
@@ -388,7 +388,7 @@ static void prvSendUpdateForJob( char * pcJobId,
 static void prvProcessJobDocument( char * pcJobId,
                                    uint16_t usJobIdLength,
                                    char * pcJobDocument,
-                                   uint16_t jobDocumentLength );
+                                   size_t jobDocumentLength );
```

Comment:
```suggestion
                                   size_t uxJobDocumentLength );
```

Should we align the coding style for size_t variables?